fenix/app/src/androidTest/java/org/mozilla/fenix/ui/robots/SettingsSubMenuAutofillRobot.kt

```diff
@@ -314,14 +410,22 @@ private val toolbarDeleteAddressButton = itemWithResId("$packageName:id/delete_a
 private val cancelDeleteAddressButton = onView(withId(android.R.id.button2)).inRoot(RootMatchers.isDialog())
 private val confirmDeleteAddressButton = onView(withId(android.R.id.button1)).inRoot(RootMatchers.isDialog())
 
+private val creditCardsSectionTitle = itemContainingText(getStringResource(R.string.preferences_credit_cards))
+private val saveAndAutofillCardsOption = itemContainingText(getStringResource(R.string.preferences_credit_cards_save_and_autofill_cards))
+private val saveAndAutofillCardsSummary = itemContainingText(getStringResource(R.string.preferences_credit_cards_save_and_autofill_cards_summary))
+private val syncCardsAcrossDevicesButton = itemContainingText(getStringResource(R.string.preferences_credit_cards_sync_cards_across_devices))
```

Comment:
I would include `credit` in the function names for consistency.

Reply:
Done !